### PR TITLE
Violation code table in README

### LINKED ISSUE #1: Add README file listing flake8 violations defined
I really like the idea of this plugin as a short term solution until black https://github.com/psf/black/issues/26 is fixed.

I would encourage you to add a README file including an example showing what the flake8 error code prefix you are using is (recommended three letter, three number format, e.g. ``ABC001`` to reduce the chance of duplicate codes between plugins). In your case, ``ISC`` as a prefix make sense to me.

### PR DESCRIPTION
**Pull request by peterjc**
This would close #1

### CODE REVIEW

**Comment by keisheiled** [comment]
I'm going to leave this until https://github.com/keisheiled/flake8-implicit-str-concat/pull/9 is merges as the scope of the package will have slightly changed

File: README.md [hunk 1]
```diff
@@ -6,8 +6,10 @@
 [![GitHub](https://img.shields.io/github/license/keisheiled/flake8-implicit-str-concat.svg)](LICENSE)
 [![Code style: Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-Forbid implicitly concatenated string literals on one line such as those
-introduced by [Black](https://github.com/psf/black/issues/26).
+This is a plugin for the Python code checking tool [flake8](http://flake8.pycqa.org/)
```
Comment (hugovk) [inline README.md]:
```suggestion
This is a plugin for the Python code checking tool [Flake8](http://flake8.pycqa.org/)
```

File: README.md [hunk 2]
```diff
@@ -38,3 +40,12 @@ All done! ✨ 🍰 ✨
...
 ```
+
+## Violation codes
+
+The plugin uses the prefix ``ISC``, short for Implicit String Concatenation.
+
+| Code   | Description                                                    |
+|--------|----------------------------------------------------------------|
+| ISC001 | implicitly concatenated string literals on one line            |
+| ISC002 | implicitly concatenated string literals over continuation line |
```
Comment (keisheiled) [inline README.md]:
```suggestion
| ISC002 | implicitly concatenated string literals over continuation line   |
| ISC003 | explicitly concatenated string should be implicitly concatenated |
```

Comment (peterjc) [reply README.md]: Rebased and included.

File: README.md [hunk 3]
```diff
@@ -6,8 +6,10 @@
 [![GitHub](https://img.shields.io/github/license/keisheiled/flake8-implicit-str-concat.svg)](LICENSE)
 [![Code style: Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-Forbid implicitly concatenated string literals on one line such as those
-introduced by [Black](https://github.com/psf/black/issues/26).
+This is a plugin for the Python code checking tool [Flake8](http://flake8.pycqa.org/)
```
Comment (keisheiled) [inline README.md]: should probably repeat `Flake8 plugin to encourage correct string literal concatenation.` here somewhere

Comment (peterjc) [reply README.md]: Updated to use the encourage wording.

**Pull request merged**